### PR TITLE
add $PATH, $GOPATH mentions

### PR DESCRIPTION
**Pull request by matkoniecz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,8 @@ This project exists to combine:
 
 [![asciicast](https://asciinema.org/a/BU00aXIcZV9bYWRko7i7LnugY.png)](https://asciinema.org/a/BU00aXIcZV9bYWRko7i7LnugY)
 
+Setup `$GOPATH` which selects where go installs packages and add `bin` directory in `$GOPATH` directory to your `$PATH`, to make `gmailctl` command available.
```
Comment (mbrt) [inline README.md]: Official Go docs already explain this quite well. I would link them instead.

```
Make sure to setup your [$GOPATH](https://golang.org/doc/code.html#GOPATH) correctly, including the `bin` subdirectory in your `$PATH`.
```

**Comment by matkoniecz** [comment]
replaced by #12

**Comment by mbrt** [comment]
Just FYI, you could have committed and pushed again. This PR would have been updated automatically, without the need for you to close it and open a new one.

**Comment by matkoniecz** [comment]
> Just FYI, you could have committed and pushed again. This PR would have been updated automatically, without the need for you to close it and open a new one.

I know, but this time I was editing via github interface and there was no access to squashing/rebasing.

**Comment by mbrt** [comment]
OK, no problem.

**Pull request closed without merging**